### PR TITLE
Kafka roles refactoring: kafka/kafka-mesos

### PR DESCRIPTION
**Pull request by Y4Rv1K**
Create two separated roles:
- kafka-mesos = role that working inside cluster and used to
create/setup/start/stop/delete kafka brokers etc.
(admin/ops tool)
- kafka = role that installing basic kafka client and can be used
inside and outside cluster (dev machines)

As result inside cluster roles kafka-mesos and kafka will be
installed on the same machine by default (no changes in cluster
behaviour)

FeatureID: Internal

### CODE REVIEW

**Comment by Y4Rv1K** [comment]
@OleksandrBerchenko please review code

File: roles/kafka-mesos/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Spark
+
+[Apache Kafka](https://kafka.apache.org) is publish-subscribe messaging rethought as a distributed commit log
+
+## Variables
+
+- `kafka_mesos_install_dir` define folder for kafka-mesos installation. Default `/usr/local/share/kafka-mesos`
```
Comment (OleksandrBerchenko) [inline roles/kafka-mesos/README.md]: Default:

File: roles/kafka-mesos/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- `kafka_scala_version`: version of scala. Default: "2.10"
+- `kafka_mesos_repo`: mesos-kafka repo URL. Default: https://github.com/mesos/kafka
+- `mesos_lib`: Specify mesos library location. Default: `/usr/local/lib/libmesos.so`
+- `zookeeper_master`: Specify FQDN for zookeeper master. Default for MI project is `zookeeper.service.consul`
+- `marathon`: Specify FQDN for marathon services. Default for MI project is `marathon.service.consul`
+- `kafka_scheduler_hostname`: hostname of kafka mesos scheduler. Default: `kafka-mesos-scheduler.service.consul`
+- `kafka_brokers`: Define number of brokers that will be used in your cluster. Default: 3
+- `kafka_broker_memory`: Specify Java memory setting for broker. Default: 512
+- `kafka_broker_heap`: Specify Java heap for broker. Default: 256 
+- `debug_enabled`: enable or dissable debug mode for kafka-mesos. Default: false
```
Comment (OleksandrBerchenko) [inline roles/kafka-mesos/README.md]: disable

File: roles/kafka/README.md [hunk 3]
```diff
@@ -5,19 +5,9 @@
 ## Variables
 
 - `kafka_install_dir`: define folder for kafka installation. Default `/usr/local/share/kafka`
```
Comment (OleksandrBerchenko) [inline roles/kafka/README.md]: Default:

**Comment by OleksandrBerchenko** [comment]
Looks good to me!  Just fix few typos.

**Comment by AndriiOmelianenko** [comment]
@Y4Rv1K @OleksandrBerchenko 
Tests passed successfully.

**Pull request merged**